packages/keyda-demo/src/lib/Input/index.js

```diff
@@ -48,6 +66,12 @@ const Input = (props) => {
 
   const handleKeyUp = useCallback(
     (e) => {
+      if (_.isEqual(keyState.inputRef, {})) {
+        keyDispatch({
+          type: 'SET_REF',
+          inputRef: inputRef,
+        });
+      }
```

Comment:
Because `keyState.inputRef === {}` doesn't work, So I used `_.isEqual()` to compare them properly.